packages/context/src/oidc/vanilla/oidc.ts

```diff
@@ -59,18 +60,19 @@ export interface StringMap {
     [key: string]: string;
 }
 
- export type OidcConfiguration = {
+export type OidcConfiguration = {
     client_id: string,
     redirect_uri: string,
-     silent_redirect_uri?:string,
-     silent_signin_timeout?:number,
+    silent_redirect_uri?:string,
+    silent_signin_timeout?:number,
     scope: string,
     authority: string,
+    authority_configuration?: AuthorizationServiceConfigurationJson,
```

Comment:
I would not exporse directly AppAuthJS type. I prefer to create a new one to protect the library from internal library change.